### PR TITLE
moved comment to new line to remove overflow-x scrollbar

### LINKED ISSUE #13352: [beta][enhancement] example code window within the instructions panel
Regular Expressions: Match Numbers and Letters of the Alphabet
http://beta.freecodecamp.com/en/challenges/regular-expressions/match-numbers-and-letters-of-the-alphabet

There is a code block that is too long in line length, it forces an overflow-y.

We can instead create a new line for the comment

This accomplishes 2 things:

-this will remove the scrollbar 
-allows everyone the opportunity to read the comment and learn what the code does

screen shot 2017-02-14 at 10 15 30 am
screen shot 2017-02-14 at 10 15 38 am

### PR DESCRIPTION
**Pull request by cdrani**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13352

#### Description

Moved comment to next line to negate the need for a scrollbar in the x direction as the text was overflowing on the desktop view.

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/regular-expressions.json [hunk 1]
```diff
@@ -305,7 +305,7 @@
         "Using the hyphen (<code>-</code>) to match a range of characters is not limited to letters. It also works to match a range of numbers.",
         "For example, <code>/[0-5]/</code> matches any number between <code>0</code> and <code>5</code>, including the <code>0</code> and <code>5</code>.",
         "Also, it is possible to combine a range of letters and numbers in a single character set.",
-        "<blockquote>let jennyStr = \"Jenny8675309\";<br>let myRegex = /[a-z0-9]/ig;<br>jennyStr.match(myRegex); // matches all letters and numbers in jennyStr</blockquote>",
+        "<blockquote>let jennyStr = \"Jenny8675309\";<br>let myRegex = /[a-z0-9]/ig;<br>jennyStr.match(myRegex);<br>// matches all letters and numbers in jennyStr</blockquote>",
```
Comment (Greenheart) [inline seed/challenges/02-javascript-algorithms-and-data-structures/regular-expressions.json]: @cdrainxv What do you think about moving the comment to the line before the regex?

**Review by Greenheart — CHANGES_REQUESTED** [review]
@cdrainxv Thanks for the fix! :blush:

File: seed/challenges/02-javascript-algorithms-and-data-structures/regular-expressions.json (hunk 1 above)
Comment (cdrainxv) [inline seed/challenges/02-javascript-algorithms-and-data-structures/regular-expressions.json]: I agree.

**Review by Greenheart — APPROVED** [review]
@cdrainxv Thanks!

This is ready to merge

**Pull request merged**